notebooks/community/model_evaluation/automl_tabular_classification_model_evaluation.ipynb

```diff
@@ -0,0 +1,1427 @@
+{
```

Comment:
<p>This gives me the following error: </p><pre class="ql-syntax" spellcheck="false">AttributeError: 'NoneType' object has no attribute 'artifacts'
</pre><p>Can you please run again and see if you get this error too? Is this code tested?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/911/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='72'/>

Reply:
<p>I tested it again. It worked fine for me. </p>

Reply:
Here is a snapshot if it: 
<img width="440" alt="evaluation_print_step" src="https://user-images.githubusercontent.com/16080687/188565462-db35b305-bdd4-4a5c-b83e-27fb1f939347.PNG">

Reply:
<p>Hmm I ran it again and didn't work. Maybe we'll need to debug that. I am running it on workbench. Are these notebook tested on local env only?</p>

Reply:
Tested them on both Workbench and Colab.